src/pages/redeem-position/RedeemPositionWrapper.tsx

```diff
@@ -23,10 +23,9 @@ export const RedeemPositionWrapper = () => {
   const { clearPosition, errors: positionErrors, position } = usePositionContext()
   const [statusTransaction, setStatusTransaction] = React.useState<Maybe<Status>>(null)
 
-  let networkId = null
+  let networkConfig = null
   if (status._type === Web3ContextStatus.Connected || status._type === Web3ContextStatus.Infura) {
-    const { networkConfig } = status
-    networkId = networkConfig.networkId
+    ;({ networkConfig } = status)
```

Comment:
The `;` is added by the linter, I think this happens because the status object is being destructured into an already declared variable